warehouse/schema.go

```diff
@@ -350,3 +348,42 @@ func getTableSchemaDiff(tableName string, currentSchema, uploadSchema warehouseu
 	}
 	return diff
 }
+
+// returns the merged schema(uploadSchema+localSchema) for all tables in uploadSchema
+func mergeUploadAndLocalSchemas(uploadSchema, localSchema warehouseutils.SchemaT) warehouseutils.SchemaT {
```

Comment:
can we call `localSchema` as `schemaInWarehousePreUpload` or something for readability. We did similar mistake last time too and its not so readable now in current code with all the different names for schemas

Reply:
Yeah 👍

Reply:
@dhawal1248 it might be easier if we store them as separate keys in metadata - year, month, day, hour. This way we can also if necessary group them in sql when doing getpendingstagingfiles etc. Also timestamp can be buggy if in future different service/codeflows parse it differently